api/python/provisioner/commands/auto_deploy.py

```diff
@@ -77,9 +96,19 @@ def run(self, nodes, **kwargs):
                 ), targets=setup_ctx.run_args.primary.minion_id
             )
 
+        logger.info("Deployment Pre-Flight Validations")
+
+        self.deployment_validations(GroupChecks.PRE_CHECKS.value)
+        logger.info("DEPLOYMENT PRE-FLIGHT CHECKS DONE")
```

Comment:
I don't think we need that log or we should add Done for all other logged staged. But it would be an unnecessary noise in logs.

Reply:
Just to make sure and test the right set of checks is completed. Deleted now.